_config.yml

```diff
@@ -0,0 +1,31 @@
+title: Project Meteor
+author: Shooting Star
```

Comment:
Do we want to keep this title/author? Maybe `Meteor Demo` with author `AICoE` or something along those lines?